pkg/windows/installer/Salt-Minion-Setup.nsi

```diff
@@ -285,6 +285,16 @@ Section "MainSection" SEC01
     File /r "..\buildenv\"
     nsExec::Exec 'icacls c:\salt /inheritance:r /grant:r "*S-1-5-32-544":(OI)(CI)F /grant:r "*S-1-5-18":(OI)(CI)F'
 
+    ; Extract portable.py script so we can fix paths in pip.exe
+    SetOutPath "$TEMP\"
+    File "..\portable.py"
+
+    ; Update python pip module with correct path.
+    nsExec::Exec '$INSTDIR\bin\python.exe "$TEMP\portable.py" -f "$INSTDIR\bin\Scripts\pip.exe" -s "..\\" -r $INSTDIR\bin\'
```

Comment:
`..\` seems too general for a search and replace in an entire executable file.

What about either:
```
-s "..\\python" -r $INSTDIR\bin\python'
```
or
```
-s "..\\python.exe" -r $INSTDIR\bin\python.exe'
```
just to be a bit safer. The second one will only work on 64-bit installs, I believe, due to the name of the exe on 32-bit being different. The first one should work on both (but is slightly less safe).

Thoughts?

Reply:
Look fine for me. The first one seems better to avoid 32/64 bits differences.

Reply:
I am fine with using the first one then.

Reply:
You could probably remove the entire section of `build_pkg.bat` where it makes those portable:
```
:: Remove the fixed path in .exe files
@echo Removing fixed path from .exe files
@echo ----------------------------------------------------------------------
"%PyDir%\python" "%CurDir%\portable.py" -f "%BinDir%\Scripts\easy_install.exe"
"%PyDir%\python" "%CurDir%\portable.py" -f "%BinDir%\Scripts\easy_install-%PyVerMajor%.%PyVerMinor%.exe"
"%PyDir%\python" "%CurDir%\portable.py" -f "%BinDir%\Scripts\pip.exe"
"%PyDir%\python" "%CurDir%\portable.py" -f "%BinDir%\Scripts\pip%PyVerMajor%.%PyVerMinor%.exe"
"%PyDir%\python" "%CurDir%\portable.py" -f "%BinDir%\Scripts\pip%PyVerMajor%.exe"
"%PyDir%\python" "%CurDir%\portable.py" -f "%BinDir%\Scripts\wheel.exe"
@echo.
```

and then modify them in the nullsoft script without specifying a search term:
```
nsExec::Exec '$INSTDIR\bin\python.exe "$TEMP\portable.py" -f "$INSTDIR\bin\Scripts\pip.exe" -r $INSTDIR\bin\'
```

Need to figure out a way to get Major and Minor versions of Python